// Copyright (c) 2020 The Orbit Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

package orbit_client_protos;

message FunctionStats {
  uint64 count = 1;
  uint64 total_time_ns = 2;
  uint64 average_time_ns = 3;
  uint64 min_ns = 4;
  uint64 max_ns = 5;
}
